ftest-toxic-reporter/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>arquillian-cube-q</artifactId>
+    <groupId>org.arquillian.cube.q</groupId>
+    <version>1.0.0.Final-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-cube-q-reporter-ftest-toxic</artifactId>
```

Comment:
I didn't mean adding it into the ftest-toxic-frontend module. I meant keeping it in the module reporter. 
Just for clarity: I'm not saying that some approach is wrong and some is correct. I'm just curious whether there are some benefits when we have it in separate module. Or if there is some issue that prevents having it in the same module...